src/Quotes/Quote.js

```diff
@@ -0,0 +1,45 @@
+import { useState, useEffect } from 'react';
+
+export function Quote() {
+  const [data, setData] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [hasError, setHasError] = useState(false);
+  const involve = {
+    headers: { 'X-Api-Key': '8aT24gv6TBZiwqX+BHiGug==VwbWI4RgUoRH1xu0' },
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await fetch('https://api.api-ninjas.com/v1/quotes?category=forgiveness', involve);
+
+        const json = await res.json();
+        setData(json[0]);
+      } catch (error) {
+        setHasError(true);
+      }
+      setIsLoading(false);
+    };
+    fetchData();
+  }, [setData, setIsLoading]); // eslint-disable-line react-hooks/exhaustive-deps
+  if (hasError) return <div className="quotes">Something went wrong!</div>;
+
+  if (isLoading) { return <div className="quotes">Loading...</div>; }
+  return (
+    <div className="quotes">
+      <article>
+        {data.quote}
+      </article>
+      <span className="author">
+        {' '}
+        {`"${data.author}"`}
+      </span>
+    </div>
+  );
+}
+
+export function Footer() {
+  return (
+    <></>
+  );
+}
```

Comment:
- In this [review link](https://github.com/samuelkinyanjui/mathmagicianone/pull/4#discussion_r1306383776), the previous reviewer asked you to remove unused code but you still have these codes displaying in your code base, can you kindly remove them as it is unprofessional 👍

Reply:
Dear Efiamotu i have used the footer to display on my page rather than deleting it. Also note it was optional comment